test/local/log.js

```diff
@@ -462,29 +462,137 @@ describe('log', () => {
   it('.summary should log an info message and call request.emitRouteFlowEvent', () => {
     const emitRouteFlowEvent = sinon.spy()
     log.summary({
-      app: {},
+      app: {
+        accountRecreated: false,
+        acceptLanguage: 'en',
+        remoteAddressChain: ['95.85.19.180', '78.144.14.50']
+      },
+      auth: {
+        credentials: {
+          email: 'quix',
+          uid: 'quid',
+        }
+      },
       emitRouteFlowEvent: emitRouteFlowEvent,
-      headers: {},
+      headers: {
+        'user-agent': ''
+      },
+      id: 'quuz',
       info: {
         received: Date.now()
       },
+      method: 'get',
       path: '/v1/frobnicate',
-      payload: {}
+      payload: {
+        reason: 'grault',
+        redirectTo: 'garply',
+        service: 'corge',
+      },
+      query: {
+        keys: true
+      }
     }, {
       code: 200,
-      statusCode: 201
+      errno: 109,
+      statusCode: 201,
+      source: {
+        formattedPhoneNumber: 'garply'
+      }
     })
 
     assert.equal(logger.info.callCount, 1)
-    assert.equal(logger.info.args[0][1].op, 'request.summary')
-    assert.equal(logger.info.args[0][1].status, 201)
-    assert.equal(logger.info.args[0][1].code, undefined)
+
+    const line = logger.info.args[0][1]
+
+    // Because t is generated using Date.now and subtracting info.received,
+    // it should be >= 0, but we don't know the exact value.
+    assert.isNumber(line.t)
+    assert.isTrue(line.t >= 0)
+
+    // Compare only known values.
+    delete line.t;
+
+    assert.deepEqual(line, {
+      op: 'request.summary',
+      status: 201,
+      errno: 109,
+      rid: 'quuz',
+      method: 'get',
+      path: '/v1/frobnicate',
+      lang: 'en',
+      agent:'',
+      remoteAddressChain: ['95.85.19.180', '78.144.14.50'],
+      accountRecreated: false,
+      uid: 'quid',
+      service: 'corge',
+      reason: 'grault',
+      redirectTo: 'garply',
+      keys: true,
+
```

Comment:
Space here for sparating Dataflow values? If so, you could put the `method` value here too.